cmd/api-errors.go

```diff
@@ -851,6 +852,11 @@ var errorCodes = errorCodeMap{
 		Description:    "Unknown tag directive.",
 		HTTPStatusCode: http.StatusBadRequest,
 	},
+	ErrTaggingOperation: {
+		Code:           "InvalidArgument",
```

Comment:
You can probably use something like `ErrNotImplemented`

Reply:
ErrTagOperation replaced with ErrNotImplemented